### PR TITLE
Masc 410 argo sync request

### PR DESCRIPTION
**Pull request by RostamKal**
A sync request can be sent before polling the status as before

### CODE REVIEW

File: argocd/sync_request copy.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (stephen-harris) [inline argocd/sync_request copy.sh:1]: Should thise file be committed? Looks like a test script :)

File: argocd/scripts/sync_and_wait.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    except ValueError as e:
+        print(f'ERROR: Decoding JSON has failed: {e}')
+        return False
+        
+    except Exception as e:
+        print(f'ERROR: Request failed: {e}')
+        return False
+
+def is_cluster_insync(endpoint, token, application, target_revision):
```
Comment (stephen-harris) [inline argocd/scripts/sync_and_wait.py:39]: I think we've duplicated some python code. Could we combine in into one python file, and then just invoke either wait or sync based on command line arguments?

File: argocd/scripts/output.json [hunk 3]
```diff
@@ -0,0 +1,377 @@
+{
+    "metadata": {
```
Comment (stephen-harris) [inline argocd/scripts/output.json:2]: Should this have been commited? Same for output-jaws-api.json?

**Review by stephen-harris — CHANGES_REQUESTED** [review]

File: argocd/scripts/sync_and_wait.py (hunk 2 above)
Comment (RostamKal) [reply argocd/scripts/sync_and_wait.py]: Good point, given this ended up being a minimal change in the end, I've now put the sync into the same job as before. No new job or scripts

**Review by stephen-harris — APPROVED** [review]

**Pull request merged**